src/Main.ts

```diff
@@ -1144,7 +1149,7 @@ export class Main {
                 room.SlackChannelName = channelInfo.channel.name;
             }
             isNew = true;
-            this.stateStorage?.trackRoom(matrixRoomId);
+            await this.stateStorage?.trackRoom(matrixRoomId);
```

Comment:
This used to be a dangling promise that wasn't waited for.
I think it's meant to be waited for.